shell/platform/linux/BUILD.gn

```diff
@@ -0,0 +1,6 @@
+# Copyright 2018 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+group("linux") {
```

Comment:
Do you also need one for "darwin"?

Reply:
That is [already present](https://github.com/flutter/engine/blob/master/shell/platform/darwin/BUILD.gn#L7).